### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by AlaaShaheen2**
- [ ] Made a console app that behaves in the same way as it is in the example given by Microverse.

- Created a app.rb file that serves as your console app entry-point.
- My app has methods to do the following:

1. List all books.
2. List all people.
3. Create a person (teacher or student, not a plain Person).
4. Create a book.
5. Create a rental.
6. List all rentals for a given person id.

- [ ] Created main.rb which does the following:

1. Present the user with a list of options to perform.
2. It lets users choose an option.
3. If needed, ask for parameters for the option.
4. Have a way to quit the app.

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @AlaaShaheen2, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a professional README however microverse changed the template, kindly consider using [the new README template provided by microverse](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  end
+
+  def list_rentals
+    print 'ID of person: '
+    person_id = gets.chomp
+    puts 'Rentals: '
+    @rentals.each do |rental|
+      if rental.person.id.to_s == person_id.to_s
+        puts "#{rental.class} #{rental.date} | Book: \"#{rental.book.title}\" rented by #{rental.person.name}"
+      end
```
Comment (codepantha) [inline app.rb:80]: - [ ] An error 🐛 occurs when trying to list the rentals associated with a given person. Kindly take a look at the error below and try to fix it 👍 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @AlaaShaheen2 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitFlow correctly
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @AlaaShaheen2,

Good job on making the changes asked by the previous reviewer 👍 

Your project is complete!  

## Highlights:
✅ You used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ App UI looks nice and all features work fine 👍 

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**